lib/text.rb

```diff
@@ -0,0 +1,42 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Text
+
+  attr_reader :sender, :client, :delivery_time
+
+  def delivery_time
+    @delivery_time = (Time.new + 3600).strftime("%H:%M")
+  end
+
+  def send_message
+  twilio_sid = 'AC6bc3bdc8a652039b67ded216da24ea1e'
+  twilio_token = 'd4411929f8c15d800b6ce5debe86e896'
+  twilio_phone_number = 441793250274
+  my_number = +447400606699
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).